src/include/wiredtiger.in

```diff
@@ -5670,11 +5677,14 @@ struct __wt_page_log_handle {
      * @param session the current WiredTiger session
      * @param page_id the page to be written
      * @param checkpoint_id the checkpoint to be written to
-     * @param is_delta is true if this is a delta, otherwise it is a full page image
+     * @param backlink_checkpoint_id the backlink checkpoint
+     * @param base_checkpoint_id the base checkpoint
+     * @param flags flags applicable to the object
      * @param buf content to be written
      */
     int (*plh_put)(WT_PAGE_LOG_HANDLE *plh, WT_SESSION *session,
-        uint64_t page_id, uint64_t checkpoint_id, bool is_delta, const WT_ITEM *buf);
+        uint64_t page_id, uint64_t checkpoint_id, uint64_t backlink_checkpoint_id,
```

Comment:
This is a lot of parameters. Should we at this point make them into a `struct` that we pass in?

Reply:
This is now done.  plh_put and plh_get now take struct args.  Things that are considered keys like page_id, checkpoint_id remain as separate args.